django-prosoul/prosoul/panels/templates/attribute-template.json

```diff
@@ -1,23 +1,24 @@
 {
     "dashboard": {
-        "id": "AWG5dj8SETw-8I6Z4dX-",
+        "id": "activity_docker-smells",
```

Comment:
I am just curious to know if I have to revert this change back. :thinking:

Reply:
I understand that this should be reverted, since the template is generic. Can you check that the new ID value isn't used anywhere? thanks

Reply:
Sure, it was replaced one more time. I would revert these changes.
Thanks.